packages/flutter_tools/lib/src/build_system/targets/android.dart

```diff
@@ -0,0 +1,124 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
would this fix https://github.com/flutter/flutter/issues/41487?

Reply:
It should already be fixed, I'll verify with this change

Reply:
would love a test :)

Reply:
Since this is already working (due to the change to build bundle), I'll add it in a separate PR here: https://github.com/flutter/flutter/issues/41487